packages/components/src/HeaderBar/HeaderBar.component.js

```diff
@@ -87,6 +87,30 @@ function Environment({ getComponent, ...props }) {
 	);
 }
 
+function CallToAction({ getComponent, t, ...props }) {
+	const actionProps = {
+		bsStyle: 'info',
+		className: 'btn-inverse',
+		label: t('HEADERBAR_ACTION', { defaultValue: 'Subscribe now' }),
```

Comment:
`label` for the Action comes from the props, so no need for default value.
And looks like `t` is not necessary.